app/src/pages/Overview.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react'
```

Comment:
I think that this PR will be cleaner if we only add the Applications page.
We can leave for other PR the changes to Overview